roles/balloonprinter/tasks/main.yml

```
@@ -0,0 +1,62 @@
+# windows updates
+#
+#
+#
+
+- name: set autologin
+  ansible.windows.win_regedit:
+    path: HKLM:\Software\Microsoft\Windows NT\CurrentVersion\Winlogon
+    name: "{{ item.name }}"
+    data: "{{ item.value }}"
+    type: "{{ item.type }}"
+  loop:
+    - { name: DefaultUserName, value: "{{ ansible_user }}", type: string }
+    - { name: DefaultPassword, value: "{{ ansible_password }}", type: string }
+    - { name: AutoAdminLogon, value: 1, type: dword }
+
+# then disable them
+- name: disable windows updates
+  ansible.windows.win_regedit:
+    path: HKLM:\Software\Policies\Microsoft\Windows\WindowsUpdate\AU
+    name: NoAutoUpdate
+    type: dword
+    data: 1
+
+# turn off screensaver/disable power saving/screen lock
+- name: turn off screen sleep/disk sleep while on AC
+  ansible.builtin.win_command: powercfg -setacvalueindex scheme_balanced {{ item }} 0
+  loop:
+    - sub_video videoidle
+    - sub_disk diskidle
+    - sub_sleep standbyidle
+
+- name: set the power plan to balanced (which is what we changed settings for)
+  ansible.builtin.win_power_plan:
+    name: balanced
+
+- name: install jre
+  chocolatey.chocolatey.win_chocolatey:
+    name: openjdk11jre
+    state: present
+
+- name: download balloon client
+  ansible.windows.win_get_url:
+    url: https://github.com/icpctools/icpctools/releases/download/v{{ balloon_version }}/balloonUtil-{{ balloon_version }}.zip
+    dest: /Users/{{ ansible_user }}/Desktop/balloonUtil-{{ balloon_version }}.zip
+
+- name: unzip balloon client
+  community.windows.win_unzip:
+    src: /Users/{{ ansible_user }}/Desktop/balloonUtil-{{ balloon_version }}.zip
+    dest: /Users/{{ ansible_user }}/Desktop/balloon
+    remote_src: true
+
+- name: copy a shortcut to launch the balloon client
+  community.windows.win_shortcut:
+    src: "c:/Users/{{ ansible_user }}/Desktop/balloon/balloon.bat"
+    dest: "c:/Users/{{ ansible_user }}/desktop/balloon.lnk"
```

```suggestion
    dest: "c:/Users/{{ ansible_user }}/Desktop/balloon.lnk"
```

Should not matter on Windows but I propose to be consistent